soc/arm64/intel_socfpga/agilex/Kconfig.defconfig.agilex

```diff
@@ -17,6 +17,6 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 	default 25000000
 
 config KERNEL_VM_SIZE
-	default 0x40000
+	default 0x180000
```

Comment:
You said this patch is to increase kernel VM size to 1MB, but 0x180000 is 1.5MB, I guess you forgot to update your commit message.

Reply:
Okay will update the comment